### ISSUE TITLE
Breaks with typescript ^3.7.X

### ISSUE THREAD

**Issue by WolverinDEV** [cevek/ttypescript#82]
Hey,
sadly `ttypescript` does not work with typescript 3.7.X or above.
It throws this error:
```
<project>\node_modules\typescript\lib\tsc.js:231
                performCompilation(sys, cb, reportDiagnostic, __assign(__assign({}, commandLine), { options: commandLineOptions }));
                                            ^

ReferenceError: __assign is not defined
    at executeCommandLineWorker (<project>\node_modules\typescript\lib\tsc.js:231:45)
    at Object.executeCommandLine (<project>\node_modules\typescript\lib\tsc.js:258:20)
    at Object.<anonymous> (<project>\node_modules\typescript\lib\tsc.js:492:4)
    at Object.<anonymous> (<project>\node_modules\ttypescript\lib\tsc.js:20:4)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)

```

My guess is because the `tsc.js` file has changed a bit.
They're now redefining some `__` functions like `__assign `.

**Comment by DoctorGester** [comment]
I'm using
    "ttypescript": "1.5.9",
    "typescript": "3.7.4",

and it works fine

**Comment by WolverinDEV** [comment]
I'll have a look later, but I'm currently quite busy (exams time^^)

**Comment by NickSeagull** [comment]
Can confirm, I'm getting the same error on TS 3.7.4, and 3.7.5

OS: Windows 10

**Comment by NickSeagull** [comment]
I have tried to remove the [`replace` call in `tsc.ts`](https://github.com/cevek/ttypescript/blob/master/packages/ttypescript/src/tsc.ts) and it works. But I guess that this is not intended.

Also, why is the minification needed?

**Comment by NickSeagull** [comment]
I see, if one removes the `replace` call, the `createProgram` doesn't get patched properly

**Comment by cevek** [comment]
@NickSeagull can you please `console.log(commandLineTsCode)` and attach it here as file?

**Comment by NickSeagull** [comment]
Ok this is embarassing, but I literally re-cloned the repo, redid everything I tried and now no error appears. Perhaps it is some issue in the lock files?

**Comment by cevek** [comment]
Do you have previous broken version?

**Comment by NickSeagull** [comment]
No sorry, I had deleted it and for some reason I cannot seem to reproduce.

**Comment by NickSeagull** [comment]
For some reason, this appeared again today, heres the log of that statement @cevek 

[log.txt](https://github.com/cevek/ttypescript/files/4871939/log.txt)

**Comment by cevek** [comment]
Fixed in ttypescript@1.5.12
